meta-oe/recipes-bsp/bolt/bolt_0.9.5.bb

```diff
@@ -12,6 +12,8 @@ SRCREV = "5a8a5866a847561566499847d46a97c612b4e6dd"
 
 S = "${WORKDIR}/git"
 
+CVE_CHECK_SKIP_RECIPE += " ${PN}"
```

Comment:
Is there normally text in this variable that we're trying to preserve? Or is the intention here really to just set the variable to `"${PN}`?

I ask because I've seen some of the OE maintainers get fussy about hedging variable assignments with `+=`.

Reply:
The [Yocto documentation](https://docs.yoctoproject.org/ref-manual/variables.html#term-CVE_CHECK_SKIP_RECIPE) says it is "The list of package names ([PN](https://docs.yoctoproject.org/ref-manual/variables.html#term-PN)) for which CVEs (Common Vulnerabilities and Exposures) are ignored."

I assume this variable could already be set by a higher-level conf file (I actually tried that earlier and it worked) or be modified by other recipes in the same fashion as I'm proposing here, so the append mechanics were intentional. However, maybe I don't fully understand the scope/lifetime of these variables to know how the values propagate. Let me know if you think this is incorrect.

Reply:
Intuitively, I would expect that variable to only ever contain one value. But I think you know more than I do at this point. So just be ready if someone on the list asks about it or pushes back.